src/components/Trips.js

```diff
@@ -12,6 +16,12 @@ function Trips() {
     setTrips(trip);
   }, []);
 
+  function handleRemoveTrip(trip) {
+    removeTripFromLocalStorage(trip);
```

Comment:
I'd ask for confirmation before removing. What if the user taps accidentally on the trashcan?

Reply:
That's a good point. I'll add that